src/compiler/emitter.ts

```diff
@@ -3187,7 +3193,9 @@ module ts {
                 }
                 else {
                     write("(");
-                    emitCommaList(node.arguments);
+                    if (node.arguments.length) {

```

Comment:
This is actually the only place we do a length check before calling `emitCommaList`. I don't feel that strongly about it, but it's one of those things that's a little odd to see out of place.